code/modules/mob/living/carbon/xenomorph/castes/jester/jester.dm

```diff
@@ -0,0 +1,148 @@
+
+/mob/living/carbon/xenomorph/jester
+	caste_base_type = /datum/xeno_caste/jester
+	name = "Jester"
+	desc = "" // This has always been wierd, xenos have two descriptions, Leave this null in favor of the castedatum desc
+	icon = 'icons/Xeno/castes/jester.dmi'
+	icon_state = "Jester Running"
+	bubble_icon = "alien"
+	health = 150
+	maxHealth = 150
+	plasma_stored = 50
+	tier = XENO_TIER_TWO
+	upgrade = XENO_UPGRADE_NORMAL
+	hud_type = /datum/hud/alien/jester
+	inherent_verbs = list(
+		/mob/living/carbon/xenomorph/proc/vent_crawl,
+	)
+	///An overlay of the caste thats picked for doppelganger, displayed ala stand style, from jojo
+	var/atom/movable/vis_obj/xeno_wounds/doppleganger_overlay/doppleganger_overlay
+	///  What type of caste this jester's doppelganger is, if it has one
+	var/datum/xeno_caste/doppelganger_caste
+
+// ***************************************
+// *********** Doppelganger Overlays
+// ***************************************
+/atom/movable/vis_obj/xeno_wounds/doppleganger_overlay
+	layer = BELOW_MOB_LAYER
+	///The xeno this overlay belongs to
+	var/mob/living/carbon/xenomorph/owner
+	alpha = 180
+
+/mob/living/carbon/xenomorph/jester/setDir(newdir)
+	. = ..()
+	if((doppelganger_caste == null))
+		return
+	switch(newdir)
+		if(NORTH)
+			doppleganger_overlay.pixel_x = 18
+			doppleganger_overlay.pixel_y = 20
+			doppleganger_overlay.layer = BELOW_MOB_LAYER
+		if(SOUTH)
+			doppleganger_overlay.pixel_x = -18
```

Comment:
doppleganger_overlay
>dopple

You might wanna go through and check all doppelganger spellings are consistent